diff -rupN a/server/subscribe_repos_test.go b/server/subscribe_repos_test.go
--- a/server/subscribe_repos_test.go	2026-10-06 19:57:03.000000000 +0000
+++ b/server/subscribe_repos_test.go	2026-10-07 20:34:07.201863760 +0000
@@ -670,7 +670,7 @@ func TestSubscribeReposFutureCursorStays
 	stop := make(chan struct{})
 	defer close(stop)
 	go func() {
-		ticker := time.NewTicker(10 * time.Millisecond)
+		ticker := time.NewTicker(5000 * time.Millisecond)
 		defer ticker.Stop()
 		for {
 			select {
